runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -804,31 +898,33 @@ public CheckPoint addAddress(long address) {
     private class StreamMetaData {
         final UUID streamId;
         CheckPoint latestCheckPoint;
-        Map<UUID, CheckPoint> checkPoints = new HashMap<>();
+        Map<UUID, CheckPoint> checkPoints = new Hashtable<> ();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651460)